README.md

```diff
@@ -33,9 +33,8 @@ Lots of things!
 
 ### Limitations
 
-- messages are deep-copied and this can be terribly inefficient for big messages like images; the current solution leverages C typesupport implementations and might benefit from a direct serialization/deserialization
```

Comment:
I hadn't pushed the changes to generalize publishing to borrowed messages yet. Now they're in, and I believe now the code is as efficient as it can be.